tests/test_external_account.py

```diff
@@ -539,7 +534,6 @@ def test_with_scopes_full_options_propagated(self):
             quota_project_id=self.QUOTA_PROJECT_ID,
             scopes=["email"],
             default_scopes=["default2"],
-            workforce_pool_user_project=None,
```

Comment:
why is this removed?

Reply:
Because the changes I've made cause it to be called without this parameter at all, and the default value of `None` gets used instead